samza-log4j2/src/main/java/org/apache/samza/logging/log4j2/StreamAppender.java

```diff
@@ -368,24 +389,24 @@ private void startTransferThread() {
             // Preserve the interrupted status for the loop condition.
             Thread.currentThread().interrupt();
           } catch (Throwable t) {
-            log.error("Error sending StreamAppender event to SystemProducer", t);
+            log.error("Error sending " + appenderName +" event to SystemProducer", t);
```

Comment:
should we add a metric to count the number of logging errors?

Reply:
added